src/ui/components/filters/geolocationcomponent.js

```diff
@@ -209,7 +196,6 @@ export default class GeoLocationComponent extends Component {
       isFilterSearch: true,
       container: '.js-yxt-GeoLocationFilter-autocomplete',
       originalQuery: this.query,
-      originalFilter: this.filter,
```

Comment:
This attribute is no longer supported by Autcomplete, right?

Reply:
I went through some of the git history including the original commit this option was added #15 and the autocomplete.hbs history https://github.com/yext/answers/commits/master/src/ui/templates/search/autocomplete.hbs.
When I set my local git back to #15 the only references to originalFilter are in geolocationcomponent.js, filterpicker.js, and filtersearchcomponent.js, where they are being put into the state, but not referenced anywhere in the .hbs . Reading through the PR it doesn't look like there was any discussion over this option. My best guess is that these were put there in anticipation of somebody overriding the component and wanting to use the value? It may be safer to keep it, but I looked through the AEB and also did not find any reference to originalFilter.